__tests__/e2e/message.spec.js

```diff
@@ -0,0 +1,18 @@
+const supertest = require('supertest');
+const app = require('../../server/app');
+
+describe('message', () => {
+    
+    it(
+        'message/',
+        async () => {
+            const response = await supertest(app).post('/message').send({jobName: 'add-tree-test', payload: {treeId: '0101010101'}});
+            expect(response.status).toBe(200); // eslint-disable-line
+            expect(response.body.job.name).toEqual('add-tree-test'); // eslint-disable-line
+            expect(response.body.job.data).toMatchObject({ // eslint-disable-line
+                treeId: '0101010101'
+            }) 
+        },
+        1000 * 30
+    )
+})
```

Comment:
@dagmawig please test the receiving, that is to say, you should subscribe the message queue name: 'add-tree-test' and make sure the message that you sent is received by the client.

like this:

```
it('test', done => {
  const client = new pgboss.client();
  client.subscribe('add-tree-test', data => {
    expect(data).toMatchObject({message:'ssss'});
    done();
   });
   // do the post

}}
```